metasmoke.py

```diff
@@ -81,12 +98,21 @@ def send_status_ping(self):
                 json_response = response.json()
                 commit_response = json_response["commit_status"]
                 if commit_response["status"] == "success":
+                    autopull_message = "Message contains 'autopull', pulling..."
```

Comment:
Looks better, but I think something was transposed here (you have `status` and then `{success}`. Also, can you move L104-107 to L103 and L109-111 to L108?